mkdocs/structure/pages.py

```diff
@@ -54,10 +54,10 @@ def __ne__(self, other):
 
     def __repr__(self):
         title = f"'{self.title}'" if (self.title is not None) else '[blank]'
-        return "Page(title={}, url='{}')".format(title, self.abs_url or self.file.url)
+        return f"Page(title={title}, url='{self.abs_url or self.file.url}')"
```

Comment:
I don't like this change.

```suggestion
        url = self.abs_url or self.file.url
        return f"Page(title={title}, url='{url}')"
```